src/components/Header.js

```diff
@@ -1,11 +1,23 @@
-import React from "react";
+import React from 'react';
+import styled from 'styled-components';
 
 const Header = () => {
   return (
-    <div className="header">
-      <h1> React and Morty </h1>
-    </div>
+    <StyledHeader>
+      <h1>React and Morty</h1>
+    </StyledHeader>
```

Comment:
Do you need the `div` here? Maybe you could just style the `h1` tag with `styled-components`